document who is expected to attend membership meetings (#237)
argoproj/argoproj: community/membership.md

A change in membership role may be approved by consensus of the Argo Project leads and approvers. By convention, role changes are discussed in a meeting with Argo Project approvers before being finalized. Role changes are generally reviewed on a quarterly basis.

> The membership meeting should be attended by project leads and by approvers who have access to merge _any_ PR in their subproject. Approvers whose access is limited by the project's OWNERS file to a particular "scope" in the repo do not attend.

Reviewer: Just a nit/suggestion to wording:

```suggestion
The membership meeting should be attended by project leads and by unscoped approvers (i.e. those who have access to merge _any_ PR in their subproject). Scoped approvers (i.e. those whose access is limited by the project's OWNERS file to a particular "scope" in the repo) do not attend.
```

> The membership meeting should be attended by project leads and by approvers who have access to merge _any_ PR in their subproject. Approvers whose access is limited by the project's OWNERS file to a particular "scope" in the repo do not attend. This keeps the meetings at a reasonable size.

Author: Should I just drop the last sentence?

> The membership meeting should be attended by project leads and by approvers who have access to merge _any_ PR in their subproject. Approvers whose access is limited by the project's OWNERS file to a particular "scope" in the repo do not attend. This keeps the meetings at a reasonable size.

Reviewer 2: IMO, it doesnt really add anything, so might as well.

> The membership meeting should be attended by project leads and by approvers who have access to merge _any_ PR in their subproject. Approvers whose access is limited by the project's OWNERS file to a particular "scope" in the repo do not attend. This keeps the meetings efficient by focusing the discussion among those who are best-positioned to evaluate new memberships and promotions.

Reviewer 3: ```suggestion
The membership meeting should be attended by project leads and by approvers who have access to merge _any_ PR in their subproject. Approvers whose access is limited by the project's OWNERS file to a particular "scope" in the repo do not attend. This keeps the meetings at a reasonable size.
```

Author: Short and sweet. Thanks!
